src/pages/[platform]/build-a-backend/auth/auth-migration-guide/index.mdx

```diff
@@ -2657,20 +2436,25 @@ This API has been deprecated: existing use cases can be migrated to a combinatio
 
 ### Example
 
-<Columns columns={2}>
-  <div>
-    **V5**
-    
-    ```
-    import { Auth } from 'aws-amplify';
+<BlockSwitcher>
+  <Block name="V6">
+    ```js
+    import {
+      fetchAuthSession,
+      getCurrentUser
+    } from 'aws-amplify/auth';
 
     const getAuthenticatedUser = async () => {
       const {
         username,
-        signInUserSession: session,
-        authenticationFlowType
-      } = await Auth.currentAuthenticatedUser();
+        authFlowType: authenticationFlowType
```

Comment:
`authFlowType` option is inside of `signInDetails` 
```suggestion
signInDetails
```

Reply:
Good catch, thanks